apps/notifications/src/panel/boot/stylesheets/main.scss

```diff
@@ -3,22 +3,22 @@
 }
 
 .wpnc__main {
-	background-color: var( --color-white );
+	background-color: var( --studio-white );
```

Comment:
"studio" has no meaning for people working on Calypso. I think it should be consistent with all other color prefixes in Calypso — `--color`.

Reply:
I mentioned this in the notes:

> All app-specific color properties are now prefixed with `--color-` while generic colors are directly outsourced to Color Studio properties (prefixed with `--studio-`).

### Code duplication

I don’t think we can afford to rename all the generic colors, because all of that would end up in the output file, duplicating the amount of code that we really need.

```scss
:root {
   /* Defined directly in Color Studio (included automatically) */
  --studio-blue-50: #2271b1;
  --studio-blue-50-rgb: 34, 113, 177;

  /* Renamed later in color schemes (duplicated lines) */
  --color-blue-50: var( --studio-color-blue-50 );
  --color-blue-50-rgb: var( --studio-color-blue-50-rgb );
}
```

The above would be repeated for every single color in our palette. We used to rename the colors by relying on SCSS color variables instead, but those were phased out.

```scss
:root {
  --color-blue-50: #{$studio-color-blue-50};
  --color-blue-50-rgb: #{hex-to-rgb( $studio-color-blue-50 )};
}
```

The introduction of the color schemes marked the beginning of our migration away from SCSS color variables to CSS color properties. We used both formats inconsistently in our stylesheets (even for theming) when we only need one. Given that CSS custom properties are the future standard, there’s no question about which one we should use.

The `--studio` prefix isn’t ideal, but the color doc in Calypso will make it clear that we’re using Color Studio—which is also an open-source tool from Automattic (and which our other projects also rely on).

### Themable properties

If you take a look at the [default color scheme](https://github.com/Automattic/wp-calypso/blob/update/colors/packages/calypso-color-schemes/src/shared/color-schemes/_default.scss), you’ll notice that all themable properties are prefixed with `--color`. This means that all `--color` properties can be overwritten by another color scheme—and that’s the main difference from the `--studio` properties, which can’t be overwritten as they’re the building blocks of all the color schemes.

---

I’m open to other suggestions as to how to make that clearer, but the truth that you caught me somebody else’s code smell. When writing CSS, we shoud rely on `--color` properties wherever possible (`--color-surface` in this case).